core/src/main/java/nl/nn/adapterframework/core/PipeLineSession.java

```diff
@@ -196,6 +280,8 @@ public String getString(@Nonnull String key) {
 			return (String) obj;
 		} else if (obj instanceof Number) {
 			return obj.toString();
+		} else if (obj instanceof Message) {
+			return ((Message) obj).asString();
```

Comment:
Voorheen werd hij gesloten, is dit niet meer de bedoeling? Misschien een commentaartje toevoegen?

Reply:
Iets wat al als Message in de PipeLineSession zit, moet niet worden gesloten want wordt misschien opnieuw gelezen.
Ik zal er een comment bij toevoegen!